virttest/libvirt_xml/vm_xml.py

```diff
@@ -3319,6 +3321,16 @@ def __init__(self, virsh_instance=base.virsh):
                                  libvirtxml=self,
                                  parent_xpath='/',
                                  tag_name='ioapic')
+        accessors.XMLAttribute(property_name='kvm_dirty_ring_state',
```

Comment:
The best way to to create a new XML class as nested one under <features>, for example, VMFeatureKvmXML for below xml. There is an enhancement we can do, but maybe in another PR. For current PR, it is fine.  
```
<features>
...
  <kvm>
    <hidden state='on'/>
    <hint-dedicated state='on'/>
    <poll-control state='on'/>
    <pv-ipi state='off'/>
    <dirty-ring state='on' size='4096'/>
  </kvm>
...
</features>
```

Reply:
@chloerh , FYI.

Reply:
@dzhengfy Agreed.